create-logux/index.d.ts

```diff
@@ -73,10 +73,13 @@ interface StateListener<S> {
 }
 
 export class LoguxVuexStore<S = any> extends VuexStore<S> {
+  constructor (options: VuexStoreOptions<S>)
+  dispatch: VuexDispatch
```

Comment:
We didn't use dispatch. It's a [part of Vuex](https://github.com/vuejs/vuex/blob/dev/types/index.d.ts#L18). Why we should add this?

Reply:
Yeap. You are extending `VuexStore` and dispatch are going to `LoguxVuexStore` signature. So we have it in API page https://logux.io/vuex-api/#loguxvuexstore-dispatch